### DISCUSSION TITLE
At import rule issue with bundle.css file

### DISCUSSION THREAD

**Discussion by jamestagal** [plentico/plenti#322] [Q&A] [answered]
Hi @jimafisk 

I am getting an issue from the following site, about the ```bundle.css``` file being ignored because of  ```@import``` rule not being defined. 

https://capital-tigers.netlify.app/ 

<img width="1916" alt="Screen Shot 2024-05-10 at 12 20 04 pm" src="https://github.com/plentico/plenti/assets/6309595/5663b12a-6fdc-4312-acc9-4242a1535b35">

The message in the console say: ```Define @import rule at the top of the stylesheet```

> An @import rule was ignored because it wasn't defined at the top of the stylesheet. Such rules must appear at the top, before any style declaration and any other at-rule with the exception of @charset and @layer.

Effective resource
bundle.css:1

I believe that the ```bundle.css``` is the minified generated css file that get output to the public dir at build time so not sure how or if I should be injecting that file back into my local css files with a ```@import``` rule? 

Is there any way to define this file in Plenti as recommended here?

Best regards,
Ben

**Comment by jimafisk** [comment, marked as answer]
> I believe that the bundle.css is the minified generated css file that get output to the public dir at build time

Yes that's correct!

> not sure how or if I should be injecting that file back into my local css files with a @import rule?

Can you explain more about what you're trying to [@import](https://developer.mozilla.org/en-US/docs/Web/CSS/@import)? This is usually used to import style rules from other valid stylesheets. Plenti should already be aggregating your scoped styles and you can define global styles in your `static` dir as well. Thanks!

**Reply by jamestagal** [reply]
Hi Jim,

Thanks. So it sounds like I can just ignore this one then...I saw this message flagged as an issue when in the code inspector tools on the site. And because I had seen the ```bundle.css``` being referred to on Google's pages peed score perviously for things like render blocking resource,  unused CSS or late network requests I wondered whether this is an issue that I should try to fix.  

Thanks for clarifying Jim. 😸
